srcpkgs/sof-bin/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'sof-bin'
+pkgname=sof-bin
+version=1.5.1
+revision=1
+archs="i686 x86_64"
```

Comment:
Why is this limited? If it's only for x86 stuff, it should at least be `i686* x86_64*` to cover musl.

Reply:
I honestly forgot musl existed while writing the template. It's been fixed.